### DISCUSSION TITLE
Whatever happened with shape names??

### DISCUSSION THREAD

**Discussion by batinic-m** [wmgeolab/geoBoundaries#2655] [Q&A] [answered]
Why are all shape names missing from the data?

**Comment by DanRunfola** [comment]
Uh oh - which dataset?

I can dig into this later today and try to roll out a quick fix.

**Reply by batinic-m** [reply]
Every?? Angola Admin2, Austria Admin2... Those are the only ones I checked, but the field seems to be completely missing.

**Reply by DanRunfola** [reply]
Let me check this now.  I just pushed 5.0, and didn't get any flags, but that may be due to the migration of the build scripts to the local environment.  Will ping back here shortly; hopefully can get this fixed tonight (tomorrow AM at the latest).

**Comment by DanRunfola** [comment, marked as answer]
Just confirming that, yes, shapeName is missing, which is an error.  It didn't get flagged because it's technically optional (i.e., we don't have it for every case), but it should still be present in most cases.
Working on a fix now (may be ~later tonight / tomorrow before everything gets pushed after I get this running).

**Reply by DanRunfola** [reply]
Boy, what an oversight.

Error is in the new builder class, line 571.  
keepCols = ["shapeGroup","shapeID","shapeType","shapeISO","geometry"]

Simply ommitted the shapeName column from the cleanup step.  Simple fix, at least.  Will ping here when the new version is deployed, thank you for flagging this.

**Reply by batinic-m** [reply]
Any progress on this?

**Reply by DanRunfola** [reply]
Should be live later this morning, I'll ping back here

**Reply by DanRunfola** [reply]
Just to follow up, ran into a few issues, aiming for by EOB today.

**Reply by DanRunfola** [reply]
All files built, just working on getting them up on to Github now.

![image](https://user-images.githubusercontent.com/7882645/210880798-7ac54b6f-ec2c-41e4-a679-3569a60fcbf4.png)


ETA: ~1.5 hours, if the scripts push through as hoped.  I have to go do some childcare but will try to check in a little later this evening to confirm everything went as planned.  I have some new badges on the front of the repo that track the github actions building things now, so if any of those turn red you'll know something went amiss (hopefulyl not :) ).

**Reply by DanRunfola** [reply]
Alright, a full build was pushed yesterday, but I was looking at the files and it looks like some were skipped during upload for an unknown reason.  I'm going to debug that one later today and see what happened.  But for now going to close this thread out (well, stop replying here :) ), as at least the shapeName issue is fixed.  

Should figure out this push issue by EOB today.   The repo may have hit a size limit, in which case I might move forward with the refresh described over in the other thread.  We'll see what's going on.

**Reply by DanRunfola** [reply]
Just to fully close the loop on this - this is now all fixed and working as expected, as of a few days ago.
